src/_about/teams.md

```diff
@@ -211,6 +211,15 @@ tooling, file repositories and API services for modders. This team is responsibl
     {% include team/Xavier.liquid %}
 </div>
 
+## Kotlin
+
+The Kotlin Team is responsible for maintaining the Quilt Kotlin Libraries (QKL), a set of libraries based off QSL and designed for use with the Kotlin programming language. These libraries provide an official way to use Kotlin on Quilt, as well as wrappers for QSL and the Minecraft codebase for easier use in Kotlin.
+
+<div class="developer team-grid">
+    {% include team/Octal.liquid %}
+    {% include team/SilverAndro.liquid %}
+</div>
```

Comment:
We'd need cards for @NoComment1105 and @PotatoPresident, but otherwise it looks good

Reply:
I've added NoComment, Potato will need a new card as he's not part of any other Quilt teams. I've sent him a message about it, but he's very busy at the moment, so I'm planning to add his card later if I don't hear from him before this is merged.